src/read.c

```diff
@@ -3614,17 +3616,20 @@ static avifBool avifDecoderItemIsAlphaAux(avifDecoderItem * item, uint32_t color
     return auxCProp && isAlphaURN(auxCProp->u.auxC.auxType);
 }
 
-// Finds the alpha item whose parent item is colorItem and sets it in the alphaItem output parameter. Returns AVIF_RESULT_OK on
-// success. Note that *alphaItem can be NULL even if the return value is AVIF_RESULT_OK. If the colorItem is a grid and the alpha
+// Finds the alpha item whose parent item is color item and sets it in the alphaItem output parameter. Returns AVIF_RESULT_OK on
+// success. Note that *alphaItem can be NULL even if the return value is AVIF_RESULT_OK. If the color item is a grid and the alpha
```

Comment:
Nit: It seems clearer to replace `color item` by `*colorItemPtr` in these two lines (3619-3620).

Also would be good to clarify that `*colorItemPtr` is an in/out parameter.

Reply:
Done.

I think the newly added sentence a the end of the paragraph makes it obvious that this is an in/out param.